jobs/build/ose/scripts/merge-and-build.sh

```diff
@@ -339,12 +378,46 @@ ose_images.sh --user ocp-build compare_nodocker --branch rhaos-${OSE_VERSION}-rh
 
 echo
 echo "=========="
+echo "Run OIT rebase"
+echo "=========="
+
+cat >"${OIT_WORKING}/sources.yml" <<EOF
+ose: ${OSE_DIR}
+jenkins: ${OPENSHIFT_JENKINS_DIR}
+openshift-ansible: ${OPENSHIFT_ANSIBLE_DIR}
+EOF
+
+${OIT_PATH} --working-dir ${OIT_WORKING} --group openshift-${OSE_VERSION} \
```

Comment:
Since I don't see a 'cd', I'll guess that a --metadata-dir is required for all invocations.

Reply:
Fixed.